src/SharedAcceptanceTests.All/ConnectionStringHelper.cs

```diff
@@ -27,7 +27,7 @@ public static string GetEnvConfiguredConnectionStringForPersistence(string table
             var connectionString = GetEnvironmentVariable(environmentVariableName);
             if (string.IsNullOrEmpty(connectionString))
             {
-                throw new Exception($"Oh no! We couldn't find an environment variable '{environmentVariableName}' with Azure Storage connection string.");
+                return "UseDevelopmentStorage=true";
```

Comment:
Is this intentional?

Reply:
It was yes, so it falls back to the emulator.